packages/firebase_snippets_app/lib/firebase_options.dart

```diff
@@ -80,7 +80,7 @@ class DefaultFirebaseOptions {
     messagingSenderId: 'TODO: use secrets or demo',
```

Comment:
Did we always have these TODO's here? I'd prefer to not use the word "secret" here, as these are configuration values and not secrets (see https://stackoverflow.com/a/37484053). While it is a common practice to *not* commit these configuration values (to prevent getting into each other's way it's often best for every dev to set up their own project), it'd be better to describe at a file-level why that is.

Reply:
Whoa, that SO answer is news to me. I shall update on a future PR.